app/src/main/java/org/mifos/selfserviceapp/ui/adapters/LoanAccountsListAdapter.java

```diff
@@ -22,12 +27,10 @@
  */
 public class LoanAccountsListAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
``` if (holder instanceof RecyclerView.ViewHolder)``` this check is redundant at [line](https://github.com/top-gun007/self-service-app/blob/fe0b14f0e3512d4fb0c0c8eaf30ab9ad7ce24568/app/src/main/java/org/mifos/selfserviceapp/ui/adapters/LoanAccountsListAdapter.java#L52) this because
```RecyclerView.ViewHolder``` is RecyclerView abstract standard class but here we have check holder is the instance of the SavingAccountsListAdapter.ViewHolder or not, So remove ```RecyclerView.ViewHolder``` to SavingAccountsListAdapter's ViewHolder.